geode-core/src/main/java/org/apache/geode/internal/tcp/TCPConduit.java

```diff
@@ -412,23 +423,24 @@ private void startAcceptor() throws ConnectionException {
    * this.bindAddress, which must be set before invoking this method.
    */
   private void createServerSocket() {
-    int p = this.port;
-    int b = BACKLOG;
+    int serverPort = this.port;
+    int connectionRequestBacklog = BACKLOG;
     InetAddress bindAddress = this.address;
 
     try {
       if (this.useNIO) {
-        if (p <= 0) {
+        if (serverPort <= 0) {
```

Comment:
Not required; but as refactoring, can we move the code under "if", "else" to its own methods.

Reply:
One of the tickets we have open is to implement SSL over NIO.  When we do that most of the conditionals in this method will go away.